packages/core/contracts/optimistic-oracle-v3/implementation/examples/DataAsserter.sol

```diff
@@ -2,17 +2,17 @@
 pragma solidity ^0.8.0;
 
 import "../../implementation/ClaimData.sol";
-import "../../interfaces/OptimisticAsserterInterface.sol";
+import "../../interfaces/OptimisticOracleV3Interface.sol";
 import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 
-// This contract allows assertions on any form of data to be made using the UMA Optimistic Asserter and stores the
+// This contract allows assertions on any form of data to be made using the UMA Optimistic Oracle V3 and stores the
 // proposed value so that it may be retrieved on chain. The dataId is intended to be an arbitrary value that uniquely
 // identifies a specific piece of information in the consuming contract and is replaceable. Similarly, any data
 // structure can be used to replace the asserted data.
 contract DataAsserter {
     using SafeERC20 for IERC20;
     IERC20 public immutable defaultCurrency;
-    OptimisticAsserterInterface public immutable oa;
+    OptimisticOracleV3Interface public immutable oa;
```

Comment:
for consistency `oa` should be renamed to `oo` as well. this applies also to other two example contracts